.github/workflows/contract-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Gateway API Contract Tests
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
the ticket must be wrong, the default branch in network-gateway is now `develop`

Reply:
`main` you mean.

Reply:
@rvelaz updated